lib/context.js

```diff
@@ -143,16 +143,18 @@ const proto = module.exports = {
     // force text/plain
     this.type = 'text';
 
+    let statusCode = err.status || err.statusCode;
```

Comment:
can we add a test case for this change?

Reply:
Yeah sure

Reply:
Added the test for statusCode change in the PR.